src/Mapper.cpp

```diff
@@ -1799,7 +1799,249 @@ namespace karto
         "found.",
         false, GetParameterManager());
   }
+  /* Adding in getters and setters here for easy parameter access */
+  
+  // General Parameters
 
+  bool Mapper::getParamUseScanMatching(){
+    return (bool)m_pUseScanMatching->GetValue();
+  }
+
+  bool Mapper::getParamUseScanBarycenter(){
+    return (bool)m_pUseScanBarycenter->GetValue();
+  }
+
+  double Mapper::getParamMinimumTravelDistance(){
+    return (double)m_pMinimumTravelDistance->GetValue();
+  }
+
+  double Mapper::getParamMinimumTravelHeading(){
+    return math::RadiansToDegrees((double)m_pMinimumTravelHeading->GetValue());
```

Comment:
ROS standardizes on radians, see http://www.ros.org/reps/rep-0103.html, so even though Karto uses degrees for their default value, I would suggest we stick with radians as the input/output of these functions